### PR TITLE
Updated the ApiSdk constructor documentation

### PR DESCRIPTION
**Pull request by Ayoseun**
## Description
Updated the ApiSdk constructor documentation comment to provide clearer guidance on the config parameter, specifying it as an Axios request config. Included an example demonstrating how the config object can be structured for various request options such as HTTP method, URL, query parameters, request body data, and headers. This enhancement improves the readability and usability of the codebase, making it easier for developers to understand and utilize the ApiSdk class.

## Related Issue

[Issue 320](https://github.com/privacy-scaling-explorations/bandada/issues/320)

## Does this introduce a breaking change?

-   No

## Other information

None

### CODE REVIEW

File: libs/api-sdk/src/apiSdk.ts [hunk 1]
```diff
@@ -20,7 +20,22 @@ export default class ApiSdk {
     /**
      * Initializes the ApiSdk object with a Supported URL or custom URL.
      * @param url Supported URL or custom URL.
-     * @param config Request config.
+     * @param config [Axios](https://axios-http.com/docs/req_config) request config type.
+     * @example config const axiosConfig = {
```
[the remark this answers is not in the document]
Comment (0xjei) [reply libs/api-sdk/src/apiSdk.ts:24]: Strongly agree with Vivian's suggestion here.

**Review by 0xjei — COMMENTED** [review]
See @vplasencia suggestions

File: libs/api-sdk/src/apiSdk.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ayoseun) [reply libs/api-sdk/src/apiSdk.ts]: Okay

[the remark this answers is not in the document]
Comment (Ayoseun) [reply libs/api-sdk/src/apiSdk.ts]: I have corrected that

**Pull request closed without merging**